src/Nettle.jl

```diff
@@ -1,7 +1,8 @@
 module Nettle
 
-if isfile(joinpath(Pkg.dir("Nettle"),"deps","deps.jl"))
-   include("../deps/deps.jl")
+const depfile = normpath(joinpath(dirname(@__FILE__), "..", "deps", "deps.jl"))
```

Comment:
true, I put it in automatically because it cuts out `folder/..` pairs but it shouldn't be necessary